src/chains/serializers.py

```diff
@@ -1,18 +1,25 @@
 from drf_yasg.utils import swagger_serializer_method
 from gnosis.eth.django.serializers import EthereumAddressField
 from rest_framework import serializers
+from rest_framework.exceptions import APIException
 
 from .models import Chain
 
 
 class GasPriceOracleSerializer(serializers.Serializer):
+    type = serializers.ReadOnlyField(default="oracle")
     url = serializers.URLField(source="gas_price_oracle_url")
     gas_parameter = serializers.CharField(source="gas_price_oracle_parameter")
     gwei_factor = serializers.DecimalField(
         source="gas_price_oracle_gwei_factor", max_digits=19, decimal_places=9
     )
 
 
+class GasPriceFixedSerializer(serializers.Serializer):
+    type = serializers.ReadOnlyField(default="fixed")
+    value = serializers.CharField(source="gas_price_fixed")
```

Comment:
Should this be `weiValue` to better represent which unit this field represents?

Reply:
Yes, better explicit than implicit :)